src/editor/Renderer.cpp

```diff
@@ -2913,6 +3116,12 @@ void Renderer::redo() {
 	redoCommand->execute();
 	redoHistory.pop_back();
 	undoHistory.push_back(redoCommand);
+	if (pickInfo.map->GetBspRender())
+	{
+		pickInfo.map->GetBspRender()->preRenderEnts();
+		updateEntConnections();
+		updateEntConnectionPositions();
+	}

```

Comment:
why is this needed? I think it should be part of a specific undo command if it's required at all.

Reply:
@wootguy I think this command update ents position ??

Reply:
@wootguy okay, I move this code to new function, and replaced all matches